src/v0/parity.rs

```diff
@@ -33,6 +33,61 @@ pub struct DoughnutV0 {
     pub signature: H512,
 }
 
+impl DoughnutV0 {
+    // Encodes the doughnut into an byte array and writes the result into a given memory
```

Comment:
Doc comments should use `///`
also referring to something in the code should be surrounded with backticks e.g. \`encode_signature\`